dnset/go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/matrixrorigin/matrixone-operator/dnset
+
+go 1.18
```

Comment:
I don't think each controller should be a separate go module, all the code of the operator can be managed in a same go module (the root go module)

Reply:
yeah, It will move to pkg/controller

Reply:
simply use the go.mod in repository root would be okay